unit-tests/post-processing/test-post-processing-filters.py

```diff
@@ -0,0 +1,560 @@
+# License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
Please rename as not the name will be:
`test-post-processing-post-processing-filters`
Let's call it `test-filters`
Same for other tests

Reply:
Done